torch/csrc/jit/codegen/cuda/arith.cpp

```diff
@@ -731,6 +745,56 @@ TensorView* clamp(TensorView* in, Val* min_val, Val* max_val) {
   return clamp(in->as<Val>(), min_val, max_val)->as<TensorView>();
 }
 
+// sum_to operator
+
+// Name of sum_to is different from NV fuser naming,
```

Comment:
1. this comment should be moved to the header (all the "interface" comments should be there)
2. the note is useful, although it seems incomplete: "to align with the operator name" ... which operator name?

Reply:
Thanks for the great suggestions. I have moved the comment to the header and added the torch operator name.